web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1368,6 +1368,47 @@ export const formatters: Formatters = {
     format: ({ updated_by }) =>
       `User [${updated_by}] failed to delete all service providers`,
   },
+  [eventCodes.OKTA_GROUPS_UPDATE]: {
+    type: 'okta.groups.update',
+    desc: 'Okta groups have been updated',
+    format: ({ added, updated, deleted }) =>
+      `[${added}] added, [${updated}] updated, [${deleted}] deleted`,
+  },
+  [eventCodes.OKTA_APPLICATIONS_UPDATE]: {
+    type: 'okta.applications.update',
+    desc: 'Okta applications have been updated',
+    format: ({ added, updated, deleted }) =>
+      `[${added}] added, [${updated}] updated, [${deleted}] deleted`,
+  },
+  [eventCodes.OKTA_SYNC_FAILURE]: {
+    type: 'okta.sync.failure',
+    desc: 'Okta synchronization failed',
+    format: () => `Okta synchronization failed`,
+  },
+  [eventCodes.OKTA_ASSIGNMENT_PROCESS]: {
+    type: 'okta.assignment.process',
+    desc: 'Okta assignment has been processed',
+    format: ({ name, source, user }) =>
+      `Okta assignment [${name}], source [${source}], user [${user}] has been successfully processed`,
+  },
+  [eventCodes.OKTA_ASSIGNMENT_PROCESS_FAILURE]: {
+    type: 'okta.assignment.process',
+    desc: 'Okta assignment failed to process',
```

Comment:
`OKTA_ASSIGNMENT_PROCESS` uses the passive voice, `OKTA_ASSIGNMENT_PROCESS_FAILURE` uses the active voice. Shouldn't they both use the same voice?

I suppose the second desc could be replaced with something like "Okta assignment processing has failed".

Reply:
Good call. I was attempting to unify these in terms of their voicing, but one or the other came off awkwardly to me. I think your suggestion is the way to go.